Configuration/StandardSequences/python/Digi_cff.py

```diff
@@ -42,6 +42,13 @@
 
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
 def _fastSimDigis(process):
+    # no need for the aliases for usual mixing
+    from Configuration.ProcessModifiers.premix_stage1_cff import premix_stage1
+    if premix_stage1.isChosen():
```

Comment:
@Dr15Jones could we extend the modifier classes somehow to allow `toModify` or `toReplaceWith` to act on ProcessModifier objects? Then one could replace the `__func` assigned to it with a dummy function.

Reply:
or one could also have multiple fastsim modifiers (fastsim, fastsim_premix1, ...) that then map more cleaning onto the existing interface

Reply:
In this case I think it would have to be something like fastsim and fastsim_notpremix1 (since we want to omit this process modifier in premix_stage1), which is a bit counterintuitive.